content/en/blog/_posts/XX-userns-enabled-by-default/index.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: blog
+title: "Kubernetes 1.33: User Namespaces enabled by default!"
+date: XXXX-XX-XX
+slug: userns-enabled-by-default
+author: >
+  Rodrigo Campos Catelin (Microsoft),
+  Giuseppe Scrivano (Red Hat),
+  Sascha Grunert (Red Hat)
+---
+
+In Kubernetes 1.33 support for user namespaces is enabled by default. This means
+that, when the stack requirements are met, pods can opt-in to use user
+namespaces. There is no need to enable any Kubernetes feature flag anymore!
+
+In this blog post we answer some common questions about user namespaces. But,
+before we dive into that, let's recap what user namespaces are and why they are
+important.
+
+## What is a user namespace?
+
+Note: Linux user namespaces are a different concept from [Kubernetes
+namespaces](/docs/concepts/overview/working-with-objects/namespaces/).
+The former is a Linux kernel feature; the latter is a Kubernetes feature.
+
+Linux provides different namespaces to isolate processes from each other. For
+example, a typical Kubernetes pod runs within a network namespace to isolate the
+network identity and a PID namespace to isolate the processes.
+
+One Linux namespace that was left behind is the [user
+namespace](https://man7.org/linux/man-pages/man7/user_namespaces.7.html). It
+isolates the UIDs and GIDs of the containers from the ones on the host. The
+identifiers in the container can be mapped to identifiers on the host in a way
+where the host UID/GIDs used for different containers never overlap.
```

Comment:
Rephrased slightly for possibly better flow

```suggestion
One Linux namespace that was left behind is the [user
namespace](https://man7.org/linux/man-pages/man7/user_namespaces.7.html). It
isolates the UIDs and GIDs of the containers from the ones on the host. The
identifiers in a container can be mapped to identifiers on the host in a way
where host and container(s) never end up in overlapping UIDs/GIDs.
```

Reply:
Isn't this saying that the container IDs don't overlap with the host? This is not what I'm saying, the text is also saying that the IDs used for containers do not overlap.

Reply:
Sorry I thought the idea was the ability to have entirely distinct set of UIDs/GIDs for both host and containers, if you wish. I'm not 100% clear what this is trying to say, is this so that containers have distinct set of UIDs/GIDs from host perspective? If that's the case, what about something like "... where host UIDs/GIDs associated with container(s) do not overlap with each other"?